gfortran-bugs/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ include(testing)
 
 project(gfortran-bug Fortran)
 
+add_compiler_test(SOURCES gfortran-20201204.f90 RUN_ONLY LABELS PR98141)
```

Comment:
I'm guessing this should be RUN_ONLY, since I expect it to compile but to segfault when run.